utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -316,18 +368,15 @@ def build(self, host_target):
         if self.args.build_lld:
             llvm_enable_projects.append('lld')
 
+        if self.args.test:
+            # LLVMTestingSupport is not built at part of `all`
+            # and is required by some Swift tests
+            build_targets.append('LLVMTestingSupport')
```

Comment:
This seems unrelated to this change?

Reply:
This is needed because with `LLVM_ENABLE_RUNTIMES` the new `runtimes` target does not have any dependencies to `LLVMTestingSupport` -- currently that is scheduled as a result of the [compiler-rt configuration](https://github.com/llvm/llvm-project/blob/be36f41ff9341c660b59320ede784314c1ece200/clang/runtime/CMakeLists.txt#L660

I prefer to apply it to the old way of building compiler-rt to limit the number of places where I check for the flags -- it should not cause any additional rebuilds there.

Reply:
However, if it is required for tests, I think that the tests should add that dependency, not compiler-rt.

Reply:
I would like to do that, but `build-script` does not schedule Swift as an external project of LLVM (like `build.ps1` does) -- so I cannot put `LLVMTestingSupport` as a dependencies of the test targets that need that.